### PR TITLE
Setting up the CI

### PR DESCRIPTION
**Pull request by szalpal**
This PR prepares the code base for internal CI. There are a lot of added lines of code, but only few of them is a new code. Majority od them is just copying files around, to accommodate to the old CI routines and the new ones. The new CI assumes, that every test in `qa` directory should start from `L0_...` prefix and has 2 files inside: `test.sh` and `setup.sh`, which define, how the test shall be performed.

FOR THE REVIEWER:
I'll mark the new files with a comment inside the review

### CODE REVIEW

File: qa/L0_inception_ensemble/test.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ex
```
Comment (szalpal) [inline qa/L0_inception_ensemble/test.sh:1]: new file

File: qa/L0_inception_ensemble/setup.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -ex
```
Comment (szalpal) [inline qa/L0_inception_ensemble/setup.sh:1]: new file

File: qa/L0_multi_input/setup.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -ex
```
Comment (szalpal) [inline qa/L0_multi_input/setup.sh:1]: new file

File: qa/L0_multi_input/test.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ex
```
Comment (szalpal) [inline qa/L0_multi_input/test.sh:1]: new file

File: qa/L0_identity/test.sh [hunk 5]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ex
```
Comment (szalpal) [inline qa/L0_identity/test.sh:1]: new file

File: qa/L0_identity/setup.sh [hunk 6]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -ex
```
Comment (szalpal) [inline qa/L0_identity/setup.sh:1]: new file

**Review by banasraf — DISMISSED** [review]

File: qa/L0_identity/model_repository/setup_identity_example.sh [hunk 7]
```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -e
```
Comment (klecki) [inline qa/L0_identity/model_repository/setup_identity_example.sh:1]: This is the same as setup.sh

File: qa/L0_identity/setup.sh (hunk 6 above)
Comment (klecki) [reply qa/L0_identity/setup.sh:1]: Why no license and endline?

File: qa/L0_identity/model_repository/setup_identity_example.sh (hunk 7 above)
Comment (klecki) [reply qa/L0_identity/model_repository/setup_identity_example.sh:1]: Looking at the other example, it looks like the setup.sh is the duplicate.

**Review by klecki — APPROVED** [review]
Will this code be duplicated between tests and examples? Or you plan to resolve it somehow?

File: qa/L0_identity/model_repository/setup_identity_example.sh (hunk 7 above)
Comment (szalpal) [reply qa/L0_identity/model_repository/setup_identity_example.sh:1]: removed the duplicate

**Review by klecki — APPROVED** [review]

**Review by banasraf — APPROVED** [review]

**Pull request merged**